### ISSUE TITLE
Settings are not permanently saved to EEPROM

### ISSUE THREAD

**Issue by inferior-minion** [bigtreetech/BIQU-BX#12]
Whenever any new settings (such as new PID settings, Z offset, etc.) are saved they are lost after powering off the printer.

**Comment by inferior-minion** [comment]
I have compiled the latest firmware on GitHub and tested on both the current compiled version and the version I compiled (screen flash fix) and confirmed this issue is present in both firmware versions.

**Comment by Dalthoraz** [comment]
Has any progress been made on this I have the same problem over a year later with a new printer.

**Comment by Rick18101965** [comment]
Nope.  I have another one in box and will see if new card in machine will
mak a difference otherwise the advice is to update main board by ethernet
tp comp....a royal pain..
